test/testcafe/spec/SelectAuthenticatorForEnroll_spec.js

```diff
@@ -88,7 +88,7 @@ test.requestHooks(requestLogger, mockEnrollAuthenticatorPassword)(`select passwo
   const enrollPasswordPage = new FactorEnrollPasswordPageObject(t);
   await t.expect(enrollPasswordPage.passwordFieldExists()).eql(true);
   await t.expect(enrollPasswordPage.confirmPasswordFieldExists()).eql(true);
-  await enrollPasswordPage.clickGoBackLink();
+  await enrollPasswordPage.clickSwitchAuthenticatorButton();
```

Comment:
This is probably wrong. And we do need `go back` link here.

Reply:
what do you mean this is probably wrong?

Reply:
Since this is `SelectAuthenticatorEnroll`, having `Sign in using something else` doesn't make sense..

Reply:
oh nvm! i didn't read the test details probably. It makes sense to have `Sign in using something else` here.. Also, I think we need to do an audit of all screens to find out if Go back is required anywhere. Only after deciding, it makes sense to clean it up.